### ISSUE TITLE
[BITV] 9.1.4.3/0.1 - Note: For text (including in focused state) under 18 points (or for bold text under 14 points), the contrast ratio between the foreground and background colors should be at least 4.5:1. For large text, the contrast ratio should be 3:1 or greater. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41846]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-6fe46b78da-v0-n1

**Comment by ShGKme** [comment]
This is a note for the issues in this report section.
